### DISCUSSION TITLE
What comes to mind when you think of the AsyncAPI community?

### DISCUSSION THREAD

**Discussion by mcturco** [asyncapi/brand#56] [Ideas]
👋  Hi! I am in the process of defining our brand values and impact, and would love some help from the community to brainstorm some ideas.

If `AsyncAPI` was a person, what are **3-5 words** that you would use to describe their _values, personality traits, or beliefs_?

Leave a comment in this discussion! 😄

**Comment by fmvilas** [comment]
Honest, always trying to help, transparent as fuck, funny, humble, inclusive, visionary.

**Comment by quetzalliwrites** [comment]
Creative and friendly; a happy details nerd that loves sharing concise and useful information.

**Reply by murari-goswami** [reply]
Goto person with details about what/who is coming in and going out :)

**Comment by Barbanio** [comment]
Energetic and non-conformist too.

**Comment by mcturco** [comment]
Thank you guys for your responses! I have taken your suggestions and created a user persona for Eve in FigJam. I just made it an open session, so by clicking [this link](https://www.figma.com/file/0wb9rbv7jd89IrWz8eGvvh/Eve%3A-Character-Persona?node-id=0%3A1) you can join in and make edits, even if you do not have a Figma account! The only thing is it can only be an open session for 24 hours, but I am leaving the FigJam file open to anyone who has the link and a Figma account after that. (Figma accounts are free, by the way! 😄 )

Excited to see everyone's feedback to this user persona and any suggestions of things that we can add/remove/change 😄
